src/App.jsx

```diff
@@ -39,20 +42,33 @@ function App() {
             console.log(frame)
             SocketClient.subscribe('/users/queue/messages', (message) => {
                 const messageContent = JSON.parse(message.body);
-                console.log(messageContent)
-                if (messageContent && messageContent.sender !== user.email){
-                    const notify = new Notification(messageContent.senderName, {
-                        icon: "src/assets/img/logo.svg",
-                        body: messageContent.content,
-                    });
+                // if (messageContent && messageContent.sender !== user.email) {
```

Comment:
Remove redundant code if not used anymore. But if will use it for future features, should comment: //TODO: ... to declare your code block usage

Reply:
Resolved